.gitignore

```diff
@@ -1,7 +1,17 @@
+local.properties
+
 .gradle
-/local.properties
-/.idea/workspace.xml
-/.idea/libraries
-.DS_Store
+/*/build
 /build
-*.iml
\ No newline at end of file
+
+# Ignore Gradle GUI config
+gradle-app.setting
+
+# Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)
+!gradle-wrapper.jar
+
+*.iml
+.idea
+
+*.mk
```

Comment:
@gscdev Should we ignore `.mk` files? It seems like `Android.mk` is already included in the upstream repository at https://github.com/CyanogenMod/android_packages_apps_Calculator/blob/cm-13.0/Android.mk